jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Jar.java

```diff
@@ -67,11 +78,71 @@
 
   @CommandLine.Option(
       names = "--from",
-      paramLabel = "<from>",
+      paramLabel = "<base-image>",
       description = "The base image to use.")
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private String from;
 
+  @CommandLine.Option(
+      names = "--jvm-flags",
+      paramLabel = "<jvm-flags>",
+      split = ",",
+      description = "JVM arguments.")
+  private List<String> jvmFlags = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--expose",
+      paramLabel = "<exposed-ports>",
+      split = ",",
+      description = "Ports to expose on container.")
```

Comment:
Example like `5000,7/dup`. I believe my example works but needs to confirm.

Reply:
Thank you! I confirmed that your example works.